src/transformState.js

```diff
@@ -6,6 +6,27 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const key of actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```

Reply:
here, you're iterating over the array of actions, not object
when iterating over object keys (using for-in loops) that's okay to give the current value name `key`